plugin/pkg/auth/authorizer/rbac/bootstrappolicy/testdata/controller-roles.yaml

```diff
@@ -504,6 +504,7 @@ items:
     - discovery.k8s.io
     resources:
     - endpointslices
+    - endpointslices/restricted
```

Comment:
just by comparison, since not very familiar with the restricted subresource, I see the endpoints/restricted has less privileges
```
  - apiGroups:
    - ""
    resources:
    - endpoints/restricted
    verbs:
    - create
 ```

Reply:
carrying this in OpenShift is odd. Why do we need it?